ariac-competitor/build_competitor_image.bash

```diff
@@ -2,8 +2,22 @@
 
 DOCKER_ARGS=""
 # Uncoment this line to rebuild without cache
+# TODO: expose this as an argument
```

Comment:
ok, we'll just have to do it for the competitor script as well as the server script